core/kernel/tee_ta_manager.c

```diff
@@ -787,6 +787,11 @@ static void dump_state(struct tee_ta_ctx *ctx)
 	struct tee_ta_session *s = NULL;
 	bool active __maybe_unused;
 
+	if (!ctx) {
+		EMSG_RAW("TA context already released");
```

Comment:
> Is this expected to happen in normal circumstances

With current implementation, no, it should not happen. This change is not mandatory. 
I rather implemented this as a safeguard because this function is related to dump and one could want  in case one wants to call this debug function.


Actually I used the verbosity level and formating as the standard TA state dump trace below.
Ok for `EMSG_RAW()`?

> (...) better not use _RAW so as to have file/line (...)
> (...) "NULL context" would be more accurate.

True, Propose to fix up with `EMSG("No TA status: null context reference")`.

Reply:
> EMSG("No TA status: null context reference")

Fine with me!